src/mobile-token/tokenService.ts

```diff
@@ -28,6 +29,8 @@ type ListTokensResponse = {
   tokens: RemoteToken[];
 };
 
+const client = createClient(API_BASE_URL, 15000);
```

Comment:
It is ok if we set this value from some config?

Reply:
you mean the timeout? Just wondering if we really need it?